src/sagemaker/jumpstart/constants.py

```diff
@@ -27,3 +27,5 @@
 JUMPSTART_DEFAULT_REGION_NAME = boto3.session.Session().region_name
 
 JUMPSTART_DEFAULT_MANIFEST_FILE_S3_KEY = "models_manifest.json"
+
+PARSED_SAGEMAKER_VERSION = ""  # this gets set by sagemaker.jumpstart.utils.get_sagemaker_version()
```

Comment:
This implementation is a bit misleading: this isn't a constant but a state variable.

how about moving this to the `jumpstart/utils.py` and prefix it with an underline so that `mypy` picks it up as an invalid import if it's used elsewhere?

```py
_sagemaker_version = ""

def get_sagemaker_version() -> str:
    if _sagemaker_version == "":
        _sagemaker_version = parse_sagemaker_version()
    return _sagemaker_version
```